ecpp

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import ConfigParser
+import sys
+import StringIO
+from verify import *
+
+#oneseventwoeight = 1728
+
+# dict with params
+params = {}
+def par_set():
+    params.update({'s':None, 'w':None, 'j':None, 't':None, 'a':None, 'b':None, 'q':None, 'r':None})
+
+# division function
+def division(n):
```

Comment:
the method is a library method, so it should be in `verify.py`, and second, it checks primality of the argument, provided the argument is small enough, so it probably should be something like `is_prime_64`